### PR TITLE
Create 0000DCA632B3FD5F.json

### PR DESCRIPTION
**Pull request by mvdswaluw**

### CODE REVIEW

File: 0000DCA632B3FD5F.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+        "gateway_conf": {
+                "gateway_ID": "dca632b3fd5e",
```
Comment (gonzalocasas) [inline 0000DCA632B3FD5F.json:3]: This ID must match exactly (including uppercasing) the name of the file.

**Review by gonzalocasas — CHANGES_REQUESTED** [review]
The ID is not correct in either the filename nor the file content, it has to be a MAC address with 4 extra characters inserted in the middle of it. Did you manually generate it? It is usually the installer that does that for you.

File: 0000DCA632B3FD5F.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
         "gateway_conf": {
-                "gateway_ID": " 0000DCA632B3FD5",
+                "gateway_ID": " 0000DCA632B3FD5F",
```
Comment (gonzalocasas) [inline 0000DCA632B3FD5F.json:3]: The space after the double quote will break the parsing of the file. 
But, I ask again: are you sure this is your MAC address? Are you installing this on a raspberry pi or another board?

Comment (mvdswaluw) [reply 0000DCA632B3FD5F.json:3]: Sorry. I'm not very familiar with Github. Yes I'm installing on a Rasberry Pi. I alread have a gateway running for a few years, but this is the first time I do it with remote config. MAC address dc:a6:32:b3:fd:5e for eth0 and dc:a6:32:b3:fd:5f for wlan0.

Comment (gonzalocasas) [reply 0000DCA632B3FD5F.json:3]: But if you are on a raspberry pi, I assume you're using the install on [this wiki](https://github.com/ttn-zh/ic880a-gateway/wiki) and in that case, the installer will tell you which MAC address you need to use, and it's one that in the overwhelming majority of cases starts with `B827EBFFEE`

**Pull request closed without merging**